project/Dependencies.scala

```diff
@@ -33,7 +33,7 @@ object Dependencies extends AutoPlugin {
     }
 
     object Kafka {
-      val kafkaVersion = "2.7.2"
+      val kafkaVersion = "2.8.1"
```

Comment:
I don't think we're quite ready for this upgrade yet, the update to 2.7 was a bit bumpier than expected even though it should be backwards compatible and generally smooth

Reply:
Agree, reverted this one.